images/windowsserver2022/scripts/setup-1.ps1

```diff
@@ -0,0 +1,4 @@
+Write-Host "verify and replace Windows system files." 
+sfc.exe /scannow
```

Comment:
I see this mentioned in the docs here: https://docs.microsoft.com/en-us/azure/virtual-machines/windows/prepare-for-upload-vhd-image

I typically see this command used as a "repair your system when something has gone wrong" step. Do we need this given we're doing a clean OS install?

Reply:
Removed